tools/AzureRM.BootStrapper/ProfileMap.json

```diff
@@ -0,0 +1,286 @@
+{
+    "latest": {
+        "AzureRM": [
+            "6.8.1"
```

Comment:
The latest on AzureRm is 6.13.1. Should we update this also just in case?

Reply:
we don't maintain the latest as no one uses it, updating it anyways